libraries/PPP/src/PPP.cpp

```diff
@@ -737,15 +647,44 @@ bool PPPClass::setBaudrate(int baudrate) {
   return true;
 }
 
-bool PPPClass::sms(const char *num, const char *message) {
-  if (_dce == NULL) {
-    return false;
+int PPPClass::getBatteryVoltage() const {
+  PPP_CMD_MODE_CHECK(-1);
+
+  int volt, bcs, bcl;
+  esp_err_t err = esp_modem_get_battery_status(_dce, volt, bcs, bcl);
+  if (err != ESP_OK) {
+    //log_e("esp_modem_get_battery_status failed with %d %s", err, esp_err_to_name(err));
```

Comment:
why are the log lines commented out?

Reply:
Well, I copied the functionality from other functions in PPP.cpp (RSSI + BER), which had them commented out:
https://github.com/espressif/arduino-esp32/blob/690bdb511d9f001e2066da2dda2c631a3eee270f/libraries/PPP/src/PPP.cpp#L515
https://github.com/espressif/arduino-esp32/blob/690bdb511d9f001e2066da2dda2c631a3eee270f/libraries/PPP/src/PPP.cpp#L534

I guess I'm not sure why those two are commented out...when none of the others in the entire file are.

That's literally the only reason I commented them out...

Reply:
LOL, OK...well...I uncommented those two lines referenced above in the latest commit...oops?

I would not expect those commands to be "frequent failures", but maybe they are problematic.  No issues on my end though.